packages/react/src/feature-link-list/FeatureLinkList.test.tsx

```diff
@@ -0,0 +1,92 @@
+import '@testing-library/jest-dom';
+import 'html-validate/jest';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { render, cleanup, screen } from '../../../../test-utils';
+import { FeatureLinkList, FeatureLinkListProps } from './FeatureLinkList';
+
+expect.extend(toHaveNoViolations);
+
+afterEach(cleanup);
+
+function renderLinkList(props?: FeatureLinkListProps) {
+	const { links, ...restProps } = props ?? {};
+	return render(
+		<FeatureLinkList
+			links={
+				links ?? [
+					{
+						label: 'Add a business with RAM',
+						description:
+							'Set up a business to act on behalf of it or invite others to. You can do this if you are responsible for this business in Relationship Authorisation Manager (RAM).',
+						href: '#',
+					},
+					{
+						label: 'Accept an invite',
+						description:
+							'Use an invite code from an email to act on behalf of a business.',
+						href: '#',
+					},
+					{
+						label: 'External link',
+						description: 'This is External link.',
+						href: 'https://www.google.com',
+						target: '_blank',
+					},
+				]
+			}
+			{...restProps}
+		/>
+	);
+}
+
+describe('FeatureLinkList', () => {
+	it('renders correctly', () => {
+		const { container } = renderLinkList({
+			links: [
+				{
+					label: 'Add a business with RAM',
+					description:
+						'Set up a business to act on behalf of it or invite others to. You can do this if you are responsible for this business in Relationship Authorisation Manager (RAM).',
+					href: '#',
+				},
+			],
+		});
+		expect(container).toMatchSnapshot();
+	});
+
+	it('renders valid HTML with no a11y violations', async () => {
+		const { container } = renderLinkList();
+		expect(container).toHTMLValidate({
+			extends: ['html-validate:recommended'],
+		});
+		expect(await axe(container)).toHaveNoViolations();
+	});
+
+	it('handles external links correctly', async () => {
+		renderLinkList();
+		const externalLink = await screen.queryByText('External link');
```

Comment:
This will pass if someone accidentally changes the link to a span, for example. A more robust approach would be to use `getByRole`. With this approach you could also remove `async` and the `toHaveTextContent` test on line 70. Testing library will give more helpful error messages by using role selectors, too.

```suggestion
		const externalLink = screen.getByRole('link', { name: 'External link , opens in a new tab' });
```